framework/tests/value/Issue867.java

```diff
@@ -0,0 +1,30 @@
+// Test case for Issue 867:
+// https://github.com/typetools/checker-framework/issues/867
+import org.checkerframework.common.value.qual.*;
+
+class Issue867 {
+    void test1() {
+        @IntVal({0, 1}) int x = 0;
+        @IntVal(0) int y = x++;
+        @IntVal(1) int w = x;
+    }
```

Comment:
Can you also add a few tests with expected errors?
E.g. what happens if you ++ x again? You should get an error as you leave the 0-1 range.
Having a few expected errors you check for makes sure that your code is catching mistakes, too.